### PR TITLE
Change SignIn UI

### PR DESCRIPTION
**Pull request by hyun132**
issue

- #9

what are changed.

- add font
- add PasswordInputTextViewForSignIn.js & EmailInputTextViewForSignIn.js
- change errorMessage in SignUpPhoneStore.js & SignUpPasswordStore & SignUpEmailStore.js
- change some file's UI in sign directory 

FYI
- Picker size in SignIn.js

### CODE REVIEW

File: app/src/view/entrance/Entrance.js [hunk 1]
```diff
@@ -77,13 +81,14 @@ class Entrance extends React.Component {
...
+                <Stack.Navigator initialRouteName="SignUpPhone">
                     <Stack.Screen
                         name="SignIn"
                         options={{
-                            title: 'Sign In',
+
+                            // title: 'Sign In',
                             headerStyle: {
-                                backgroundColor: Colors.primary,
+                                backgroundColor:"#f9f9f9" ,
```
Comment (jinsumin) [inline app/src/view/entrance/Entrance.js:91]: COLORS.MAIN_COLOR

**Review by jinsumin — COMMENTED** [review]
LGTM

**Review by sanghyunJ — APPROVED** [review]
LGTM

File: app/src/view/entrance/Entrance.js (hunk 1 above)
Comment (hyun132) [reply app/src/view/entrance/Entrance.js:91]: 수정완료. 다음 커밋시 반영하겠습니다

**Pull request merged**